### PR TITLE
Add missing core theming documentation

### LINKED ISSUE #2279: Add missing content to the developer docs theming guide
@voroyam pointed out that some of the content in the original (RsT) theming guide is missing from the current version of the developer docs theming guide. Please migrate it from https://github.com/owncloud/documentation/blob/5f32f2ae3c3f4b8e45fc034bc0dbe5d41948640f/developer_manual/core/theming.rst#L50.

## What Type Of Content Change Is This? (Optional)
- [ ] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [X] Missing Content
- [ ] Bug Fix to Existing Content

## Which Manual Does This Relate To? (Optional)
- [ ] Admin Manual
- [X] Developer Manual
- [ ] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [ ] Desktop Client
- [ ] Other

### PR DESCRIPTION
**Pull request by settermjd**
This fixes #2279 by adding the missing theming documentation from the old, documentation, repository. It adds the supporting images as well.

### CODE REVIEW

**Comment by mmattel** [comment]
adding @ChrisEdS fyi.
You may want to add the final http link of this document when merged (pls use "latest" and not any release number in case) to the readme in theme-example.

**Comment by settermjd** [comment]
> adding @ChrisEdS fyi.
> You may want to add the final http link of this document when merged (pls use "latest" and not any release number in case) to the readme in theme-example.

Sorry, what?

**Comment by mmattel** [comment]
@settermjd not you...,
it is a hint for @ChrisEdS that he may link this (when merged) in readme.md in theme-example repro

**Comment by settermjd** [comment]
> @settermjd not you...,
> it is a hint for @ChrisEdS that he may link this (when merged) in readme.md in theme-example repro

Thanks for the clarification.

File: modules/developer_manual/pages/core/theming.adoc [hunk 1]
```diff
@@ -224,6 +225,124 @@ NOTE: When using custom filetype icons in a custom theme, it is necessary to run
...
+=== How to Change the Header
+
+image::core/theming/4_oc_header.png[The default ownCloud header]
+
+The image above is the default ownCloud header.
+
+image::core/theming/5_cu_header.png[Set a custom ownCloud header image]
+
+You can change it to a custom color with a custom logo. 
+Search for body-public `#header` in styles.css.
```
Comment (ChrisEdS) [inline modules/developer_manual/pages/core/theming.adoc]: Search for `#body-public #header` in `core/css/styles.css`.

Comment (settermjd) [reply modules/developer_manual/pages/core/theming.adoc]: Fixed. Thanks for spotting that.

**Pull request merged**